buildings/gui/alter_building_relationships.py

```diff
@@ -343,16 +350,21 @@ def multi_selection_changed(self):
         # error msg when more than one set of matched or related set are selected
         if has_multi_set:
             self.message_bar_edit.pushMessage('Multiple matched or related sets selected, can only unlink one at a time.')
-        # switch botton
+        # switch button
         if has_matched or has_related:
             self.btn_unlink.setEnabled(True)
         elif has_added and has_removed:
             self.switch_btn_match_and_related()
+        elif has_added and not has_removed:
+            self.enable_delete()
```

Comment:
buttons have been reset at the function top `reset_buttons()` so just need to `self.btn_delete.setEnabled(True)` and you can delete the `enable_delete` function